std/internal/entropy/common.d

```diff
@@ -0,0 +1,181 @@
+// Written in the D programming language.
+
+/+
+    Non-backend-specific functionality.
+
+    License:   $(HTTP www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
+    Authors:   Elias Batek
+    Source:    $(PHOBOSSRC std/internal/entropy/common.d)
+ +/
+module std.internal.entropy.common;
+
+@nogc nothrow:
+
+///
+enum EntropySource
+{
+    /// Try supported sources one-by-one until one is available.
+    /// This exists to enable the use of this the entropy library
+    /// in a backwards compatibility way.
+    tryAll = -1,
+
+    /// Always fail.
+    none = 0,
+
+    /// `/dev/urandom`
+    charDevURandom = 1,
+
+    /// `/dev/random`
+    charDevRandom = 2,
+
+    /// `getrandom` syscall wrapper
+    getrandom = 3,
+
+    /// `arc4random`
+    arc4random = 4,
+
+    // `getentropy`
+    getentropy = 5,
+
+    /// Windows legacy
+    cryptGenRandom = 6,
+
+    /// Windows modern
+    bcryptGenRandom = 7,
+}
+
+///
+enum EntropyStatus
```

Comment:
ditto

Reply:
I’ve decided to disable this style check: <https://github.com/dlang/phobos/pull/10748/commits/cfcb0ced23a6f3168e836c647a72e6f2069e75f9>
Reasoning: <https://github.com/dlang/phobos/pull/10748#pullrequestreview-2810627921>